styles/styles.css

```diff
@@ -0,0 +1,15 @@
+h1 {
+    color: #000000;
+    text-align: center;
+
+}
+
+.support {
```

Comment:
The class name is not suitable for all being used cases. Choose a common name like "offer", "tile" etc.